bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js

```diff
@@ -161,6 +161,14 @@ pimcore.element.selector.asset = Class.create(pimcore.element.selector.abstract,
                             text: t('remove'),
                             iconCls: "pimcore_icon_delete",
                             handler: function (index, item) {
+
+                                if(this.parent.multiselect) {
+                                    var resultPanelStore = this.resultPanel.getStore();
+                                    var elementId = this.selectionStore.getAt(index).id;
+    
+                                    resultPanelStore.findRecord("id", elementId).set('asset-selected', false);
```

Comment:
```suggestion
                                    record = resultPanelStore.findRecord("id", elementId)
                                    if(record) {
                                        resultPanelStore.findRecord("id", elementId).set('asset-selected', false);
                                    }
                                    
```

Otherwise there will be an error, when you try to remove a element which is not displayed in the grid anymore. 
E.g.
 1.) Search for image, select an image
 2.) Search for video, select video
 3.) Remove image from selection panel on the right side